.github/workflows/release.yml

```diff
@@ -29,9 +29,9 @@ jobs:
       id: build_package
       run: make package
     - name: Upload package
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
```

Comment:
(question) is there a reason we are building the parser in the test workflow (vs the release workflow)?

Reply:
The test workflow runs per OS in the matrix, while the release workflow only runs once and merges the resulting artifacts together. Our test processes do a build anyway to make sure the binaries continue to work